pages/index.js

```diff
@@ -1,7 +1,11 @@
+import React from "react";
+import Link from "next/link";
+
 export default function HomePage() {
   return (
     <div>
-      <h1>Hello from Next.js</h1>
+      <h1>🐾 PfotenPortal 🐾</h1>
+      <Link href="/animals">Zur Pfotenseite </Link>
     </div>
```

Comment:
- Ach schau mal an, hier ist sie, die Function Declaration 🥳
-Instead of using a generic div-element, there is a semantic HTML element that wraps the main content of the page. Which element could this be? 😁